### PR TITLE
fix: add Clusck 2.0

### LINKED ISSUE #37: 🚀 [Illustration] - Clusck 2.0
### Link of the Illustration

https://www.ls.graphics/illustrations/clusck-2-0-illustration-kit

### Category of The Illustration

Paid

### Formats Available

Format
- Sketch
- Adobe Illustrator
- Adobe Xd
- Figma
- SVG
- Affinity
- PNG

### Author

LS Graphics

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects/blob/main/CONTRIBUTING.md) of this project.

### PR DESCRIPTION
**Pull request by at-the-vr**
resolves #37 

## Name of Illustration

[Clusck 2.0](https://www.ls.graphics/products/clusck-2-0-illustration-kit)

 

 

## Illustration Image

 

---

## Checklist

- [x] Name the PR fix: add Clusck 2.0.
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] You've linked the PR to the correct issue.
- [x] I have read the [Contribution Guidelines](https://github.com/Anmol-Baranwal/Awesome-Illustrations-4Projects/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Anmol-Baranwal** [comment]
@PiseAtharva 
Please do these minor changes in a single commit. Then I will merge the PR.

**Comment by at-the-vr** [comment]
got it, tomorrow evening 👍

**Comment by Anmol-Baranwal** [comment]
@PiseAtharva 

Just update the readme with the above changes in the same branch, don't create a new branch or a new PR.

It will automatically be displayed here. I will review when you have pushed the commit with above changes.

**Comment by at-the-vr** [comment]
Hey Anmol, I tried doing the way you were asking for, while it does not seems to work for me (the commits just keep on piling up in the development branch as you can see) nonetheless, changes have been updated to display Clusck2.0 clearly.

File: README.md [hunk 1]
```diff
@@ -337,6 +338,7 @@ Improve your projects using these awesome illustrations. See a sample <a href="h
...
 | <a href="https://growwwkit.com/illustrations/circuit/"><img width='700' src='https://user-images.githubusercontent.com/74038190/195027164-30e36ac4-4707-49b4-81bd-beeea4009e1f.png' alt="Circuit_paid_illustrations" /></a> | [Circuit](https://growwwkit.com/illustrations/circuit/)  <br><br> 💰 Paid  <br><br>Format <br> • Ai<br> • SVG<br> • PNG<br> • PDF<br><br> ✌️Author <br> Growww |
+| <a href="https://www.ls.graphics/products/clusck-2-0-illustration-kit"><img width='700' src= 'https://user-images.githubusercontent.com/88548999/217832206-b8d226c3-081d-4af6-933e-3174b40c2ee2.png' alt='Clusck2.0_paid_illustrations'/></a> | [Clusck 2.0](https://growwwkit.com/illustrations/circuit/)  <br><br> 💰 Paid  <br><br>Format <br> • Sketch<br> • Adobe Illustrator<br> • Adobe Xd<br> • Figma<br> • SVG<br>• Affinity<br>• PNG<br><br> ✌️Author <br> Ls Graphics |
```
Comment (Anmol-Baranwal) [inline README.md]:
```suggestion
| <a href="https://www.ls.graphics/products/clusck-2-0-illustration-kit"><img width='700' src= 'https://user-images.githubusercontent.com/88548999/217832206-b8d226c3-081d-4af6-933e-3174b40c2ee2.png' alt='Clusck2.0_paid_illustrations'/></a> | [Clusck 2.0](https://www.ls.graphics/products/clusck-2-0-illustration-kit)  <br><br> 💰 Paid  <br><br>Format <br> • Sketch<br> • Adobe Illustrator<br> • Adobe Xd<br> • Figma<br> • SVG<br>• Affinity<br>• PNG<br><br> ✌️Author <br> Ls Graphics |
```

Fix the link to the correct illustration:

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Anmol-Baranwal) [reply README.md]: I have suggested the changes.
Correct the above mentioned link.

Comment (at-the-vr) [reply README.md]: Thanks for pointing that one, just updated the url

**Review by Anmol-Baranwal — APPROVED** [review]
Well done !!

**Comment by Anmol-Baranwal** [comment]
Appreciate your contribution :)

**Pull request merged**